positioning.gemspec

```
@@ -28,8 +28,14 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
-  spec.add_dependency "activesupport", ">= 6.1"
-  spec.add_dependency "activerecord", ">= 6.1"
+  spec.add_dependency "activesupport", ">= 4.2" # Decreased to enable compatibility
+  spec.add_dependency "activerecord", ">= 4.2" # Decreased to enable compatibility
```

RuboCop (1.76.1)

[convention] Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `activerecord` should appear before `activesupport`.

(Gemspec/OrderedDependencies)